features/custom_stack_trace.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Custom stack trace
+
+  Scenario: Error.prepareStackTrace override
+    Given a file named "features/a.feature" with:
+      """
+      Feature: Some feature
+        Scenario: Some scenario
+          Given I override Error.prepareStackTrace
+      """
+    Given a file named "features/step_definitions/cucumber_steps.js" with:
+      """
+      var cucumberSteps = function() {
+        this.When(/^I override Error.prepareStackTrace$/, function() {
+          Error.prepareStackTrace = function() {
+            return 'Custom message';
+          }
```

Comment:
this doesn't fail for me on master. I would expect you to need to override it before `this.When()` gets called as that is when the stacktrace is captured.